### PR TITLE
Express API books - Emma

### PR DESCRIPTION
**Pull request by emmapre**

### CODE REVIEW

**Comment by emmapre** [comment]
I had copypasted the wrong link in the readme. Here is the link to the deploy https://emma-books-api.herokuapp.com/

File: server.js [hunk 1]
```diff
@@ -24,7 +16,40 @@ app.use(bodyParser.json())
...
+    books = books.filter((item) => item.authors.toLowerCase().includes(author.toLowerCase())
+    )
+  }
+
+  //find all titles that include
+  if (title) {
+    books = books.filter((item) => item.title.toString().toLowerCase().includes(title.toLowerCase())
+    )
+  }
+
```
Comment (KarolCislowski) [inline server.js:39]: Great functionality! 💯

File: README.md [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This is a book API built in Node using Express.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+The task was to create an API with two endpoints. I used .filter() to be able to filter all the books by a certain author or by title with queries and .find() to make the id endpoint to be able to find a single books. If I had more time I would like to make more queries and pagination.
```
Comment (KarolCislowski) [inline README.md:7]: Try to not only tell what you did it but also how to use it.
endpoints:
-/books -explain
-/books?author=xyz -filters...
-/books?title=xyz -filters...
-/books/:id -explain

File: server.js [hunk 3]
```diff
@@ -24,7 +16,40 @@ app.use(bodyParser.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Endpoints: /books, /books/id')
```
Comment (KarolCislowski) [inline server.js:19]: If you want to make it looks nicer, you can create an HTML file and serve it to the user :) 
Maybe you could add info about available queries here? :)

File: server.js [hunk 4]
```diff
@@ -24,7 +16,40 @@ app.use(bodyParser.json())
...
+    )
+  }
+
+  //find all titles that include
+  if (title) {
+    books = books.filter((item) => item.title.toString().toLowerCase().includes(title.toLowerCase())
+    )
+  }
+
+  res.json(books)
```
Comment (KarolCislowski) [inline server.js:40]: Here you could provide some sorting if you had some more time :)

**Pull request closed without merging**